lib/bindings/src/breez_liquid_sdk.udl

```diff
@@ -245,6 +245,7 @@ enum PaymentError {
 
 dictionary Config {
     string boltz_url;
+    string? referral_id;
```

Comment:
We don't want to expose that in the config. 
Also let's remove the boltz_url from the config since now we get it from the remote server.